### PR TITLE
New check: error if "editor lock" is set

### PR DESCRIPTION
**Pull request by ibnbr**
It's only looking FUGR and PROG objects type.

### CODE REVIEW

**Comment by larshp** [comment]
closes #110

File: src/checks/zcl_aoc_check_95.clas.abap [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  ENDMETHOD.
+
+
+  METHOD run.
+    DATA lv_edtx  TYPE reposrc-edtx.
+
+    SELECT SINGLE edtx
+      FROM reposrc
+      INTO lv_edtx
+      WHERE progname = object_name
```
Comment (larshp) [inline src/checks/zcl_aoc_check_95.clas.abap]: did you test this okay for function groups? I guess the object_name does not match the include name?

Comment (ibnbr) [reply src/checks/zcl_aoc_check_95.clas.abap]: You're right, I adjusted to use PROGRAM_NAME, in my tests it worked.

**Comment by ibnbr** [comment]
Done.

Sorry about lint errors, SE24 always change identation and pretty printer. 😢

**Comment by larshp** [comment]
dont worry

**Pull request merged**